src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/UnsupportedTypeConverterFactory.cs

```diff
@@ -23,6 +23,8 @@ public override bool CanConvert(Type type)
                 type == typeof(SerializationInfo) ||
                 type == typeof(IntPtr) ||
                 type == typeof(UIntPtr) ||
+                // Exlude delegates.
```

Comment:
nit: The "Exlude delegates" comment doesn't seem useful. If we keep it, there's also a typo

Reply:
👍 I really just wanted some visual separation from the previous set of types. Will clean up in main/7.0+ in follow up PR.